core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -707,6 +711,10 @@
    */
   public static final char BIG_INT_MEASURE = 'd';
   /**
+   * BOOLEAN_VALUE_MEASURE
+   */
+  public static final char BOOLEAN_MEASURE = 'a';
```

Comment:
This is not required. Others are for backward compatible only.

Reply:
ok, I have removed it.